controllers/operands/ssp.go

```diff
@@ -277,6 +277,23 @@ func getCommonDicts(list []hcov1beta1.DataImportCronTemplateStatus, crDicts map[
 	return list
 }
 
+func updateStringMap(original *map[string]string, updates map[string]string) {
+	if original == nil {
+		return
+	}
+	if len(updates) == 0 {
+		return
+	}
+
+	if *original == nil {
+		*original = map[string]string{}
+	}
+
+	for key, value := range updates {
+		(*original)[key] = value
+	}
+}
+
```

Comment:
please check the standard library `maps` package. I believe there are some built-in solutions for this.

Reply:
I've added a call to `maps.Copy()` but it does not create the map if it is `nil`.

Reply:
I know, but then you only need to check this case, and then run `maps.Copy()` or `make(map[string]string` if it's nil

Reply:
done